libtelemetry/src/main/java/com/mapbox/services/android/telemetry/MapboxTelemetry.java

```diff
@@ -35,22 +38,41 @@
   private boolean isOpted = false;
   private boolean serviceBound = false;
 
+  private static final String EVENTS_USERAGENT = "MapboxEventsAndroid/";
```

Comment:
NIT The IDE warns us that might be a typo in `EVENTS_USERAGENT`. Could we separate `USERAGENT` with an underscore into `USER_AGENT` (Java's constants standard)? Same for the rest of the constants 👇